setup.py

```diff
@@ -31,6 +31,9 @@
         'pytest>=3.0.0',
         'urllib3',
     ],
+    extras_require={
+        'xdist': ['pytest-xdist>=2.1.0'],
```

Comment:
please reflect this in the test requirements then, as apparently it was installing the older version or why is it not catching the incompatibility ATM?